sdks/python/apache_beam/io/fileio.py

```diff
@@ -530,15 +530,22 @@ def _rename_file(rename_op):
         channel_factory.rename(old_name, final_name)
       except IOError as e:
         # May have already been copied.
-        exists = channel_factory.exists(final_name)
+        try:
+          exists = channel_factory.exists(final_name)
+        except Exception as exists_e:  # pylint: disable=broad-except
+          logging.warning('Exception when invoking channel_factory.exists(): '
```

Comment:
This message isn't likely to be actionable without knowing more about the factory or the names involved. Can you add any detail without leaking information?